src/test/java/bral/museum/RijksServiceTest.java

```diff
@@ -0,0 +1,82 @@
+package bral.museum;
+
+import bral.museum.json.ArtObject;
+import bral.museum.json.ArtObjects;
+import org.junit.jupiter.api.Test;
+import com.andrewoid.ApiKey;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class RijksServiceTest {
+
+    @Test
+    public void getCollectionByPage()
+    {
+        // given
+        ApiKey apiKey = new ApiKey();
+        RijksService service = new RijksServiceFactory().getService();
+
+        // when
+        ArtObjects artObjectsResponse = service.getCollectionByPage(
+                apiKey.get(),
+                1
+        ).blockingGet();
+
+        // then
+        ArtObject artObject = artObjectsResponse.artObjects.get(0);
+        assertNotNull(artObject.longTitle);
```

Comment:
When I told you to change this for `longTitle`, I mean that you should change it everywhere that you are calling `assertNotNull(""`